exercises/J-array-get-set/exercises2.js

```diff
@@ -7,6 +7,8 @@
 */
 
 let numbers = [1, 2, 3]; // Don't change this array literal declaration
+numbers[numbers.length] = 4;
+numbers[0] = 1;
```

Comment:
Even though this code works, the right way to add something at the end of the array is using prototypes methods and in this case, you need to use **.push()**